airflow/jobs.py

```diff
@@ -2135,13 +2135,7 @@ def heartbeat_callback(self, session=None):
         ti = self.task_instance
         if ti.state == State.RUNNING:
             self.was_running = True
-            fqdn = socket.getfqdn()
-            if fqdn != ti.hostname:
-                logging.warning("The recorded hostname {ti.hostname} "
-                                "does not match this instance's hostname "
-                                "{fqdn}".format(**locals()))
-                raise AirflowException("Hostname of job runner does not match")
-            elif not self._is_descendant_process(ti.pid):
+            if not self._is_descendant_process(ti.pid):
```

Comment:
you just removed that check :), can we put a comment around it, lets actually start putting a comment like
########### BEGIN LYFT EDIT

#### END

This will help us in merging